src/test/java/com/dnd/runus/application/challenge/ChallengeServiceTest.java

```diff
@@ -0,0 +1,90 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static com.dnd.runus.global.constant.TimeConstant.SERVER_TIMEZONE_ID;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class ChallengeServiceTest {
+
+    @Mock
+    private RunningRecordRepository runningRecordRepository;
+
+    @InjectMocks
+    private ChallengeService challengeService;
+
+    @DisplayName("어제 기록이 있는경우 챌린지 리스트 조회 : 챌린지 name에 '어제'값이 포함한 값이 있어야함")
+    @Test
+    void getChallengesWithYesterdayRecords() {
+        // given
+        Member member = new Member(MemberRole.USER, "nickname1");
+        OffsetDateTime todayMidnight = LocalDate.now(SERVER_TIMEZONE_ID)
+                .atStartOfDay(SERVER_TIMEZONE_ID)
+                .toOffsetDateTime();
+        OffsetDateTime startTime = todayMidnight.minusDays(1).plusHours(4);
+        OffsetDateTime endTime = startTime.plusMinutes(30);
+
+        RunningRecord record = RunningRecord.builder()
+                .member(member)
+                .startAt(startTime)
+                .endAt(endTime)
+                .route(List.of(new Coordinate(123, 34), new Coordinate(123.5, 38)))
+                .emoji(RunningEmoji.VERY_GOOD)
+                .location("location name")
+                .distanceMeter(10)
+                .duration(Duration.ofSeconds(1_800))
+                .calorie(100)
+                .averagePace(new Pace(6, 0))
+                .build();
+
+        given(runningRecordRepository.hasByMemberIdAndStartAtBetween(
+                        member.memberId(), todayMidnight.minusDays(1), todayMidnight))
+                .willReturn(true);
+
+        // when
+        List<ChallengesResponse> challenges = challengeService.getChallenges(member.memberId());
+
+        // then
+        assertTrue(challenges.stream().anyMatch(c -> c.name().contains("어제")));
+    }
+
+    @DisplayName("어제 기록이 없는 경우 챌린지 리스트 조회 : 챌린지 name에 '어제'값이 포함한 값이 없어야함")
+    @Test
+    void getChallengesWithoutYesterdayRecords() {
+        // given
+        Member member = new Member(MemberRole.USER, "nickname1");
+        OffsetDateTime todayMidnight = LocalDate.now(SERVER_TIMEZONE_ID)
+                .atStartOfDay(SERVER_TIMEZONE_ID)
+                .toOffsetDateTime();
+
+        given(runningRecordRepository.hasByMemberIdAndStartAtBetween(
+                        member.memberId(), todayMidnight.minusDays(1), todayMidnight))
+                .willReturn(false);
+
+        // when
+        List<ChallengesResponse> challenges = challengeService.getChallenges(member.memberId());
+
+        // then
+        assertTrue(challenges.stream().anyMatch(c -> !c.name().contains("어제")));
```

Comment:
"어제"라는 단어가 없어야 한다면, 전부 없는지 확인해야 할 것 같아요

```suggestion
        assertTrue(challenges.stream().noneMatch(c -> c.name().contains("어제")));
```